RedirectApplication/RedirectApplication/Controllers/ValuesController.cs

```diff
@@ -39,8 +42,11 @@ public string Get(int id)
         }
 
         // POST api/values
-        public void Post([FromBody]string value)
+        public void Post(HttpRequestMessage request)
         {
+            var someText = request.Content.ReadAsStringAsync().Result;
+            var reader = new JsonTextReader(new StringReader(someText));
+            PostJson content = JsonSerializer.CreateDefault().Deserialize<PostJson>(reader);
```

Comment:
So again, it should be in another tickets PRs https://github.com/hwdtech/redirect/pull/7#discussion_r84063406